packages/view/backend/sirius-components-view-gantt/src/main/resources/model/gantt.genmodel

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unless I missed something, you have forgot to generate the view builder part for the gant view model, while it does not prevent the code from being integrated, that's a necessary part to maintain a view based representation. Please open an issue to contribute it (and maintain it when you'll change anything in the view metamodel) soon